setup.py

```diff
@@ -25,35 +25,34 @@
 Topic :: Software Development :: Testing\
 """
 
-setup(name="compare-locales",
-      version=version,
-      author="Axel Hecht",
-      author_email="axel@mozilla.com",
-      description='Lint Mozilla localizations',
-      long_description=long_description,
-      long_description_content_type='text/markdown',
-      license="MPL 2.0",
-      url="https://github.com/mozilla/compare-locales",
-      classifiers=CLASSIFIERS.split("\n"),
-      platforms=["any"],
-      python_requires='>=3.7, <4',
-      entry_points={
-        'console_scripts':
-        [
-            'compare-locales = compare_locales.commands:CompareLocales.call',
-            'moz-l10n-lint = compare_locales.lint.cli:main',
+setup(
+    name="compare-locales",
+    version=version,
+    author="Axel Hecht",
+    author_email="axel@mozilla.com",
```

Comment:
Should we change these, given the email is not valid anymore and you're maintaining this project?

Reply:
Probably, yes. I'll do so in a separate PR, though.